dist/formio.utils.js

```diff
@@ -760,76 +748,9 @@ http://ricostacruz.com/cheatsheets/umdjs.html
         }
       }
       return current; // Last
-
```

Comment:
Why was all of this removed?  You need to delete your ```node_modules``` folder, ```npm install``` and then rebuild and recommit.